src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,98 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.24.0.dev3 (12/13/2019)
+------------------------
+
+New Features
+~~~~~~~~~~~~
+
+* A rule to create a pex from a target closure. (#8811)
+  `PR #8811 <https://github.com/pantsbuild/pants/pull/8811>`_
+
+* Add Flake8 linter to V2 (#8791)
+  `PR #8791 <https://github.com/pantsbuild/pants/pull/8791>`_
+
+* Support working_directory in EPRs. (#8784)
+  `PR #8784 <https://github.com/pantsbuild/pants/pull/8784>`_
+
+* Make v2 binary goal pay attention to --pants-distdir (#8775)
+  `PR #8775 <https://github.com/pantsbuild/pants/pull/8775>`_
+
+* Fix V2 binary to work with multiple targets (#8781)
+  `PR #8781 <https://github.com/pantsbuild/pants/pull/8781>`_
+
+Bugfixes
+~~~~~~~~
+
+* Fix graph ambiguity of using multiple linters/formatters in V2 (#8801)
+  `Issue #8722 <https://github.com/pantsbuild/pants/issues/8722>`_
+  `PR #8801 <https://github.com/pantsbuild/pants/pull/8801>`_
+
+* Fix registration of unions that are split across multiple backends (#8810)
+  `PR #8810 <https://github.com/pantsbuild/pants/pull/8810>`_
+
+* Fix enginedisplay printing (#8799)
+  `PR #8799 <https://github.com/pantsbuild/pants/pull/8799>`_
+
+* Add cherry-picked commit to 1.23.0 (#8804)
+  `PR #8804 <https://github.com/pantsbuild/pants/pull/8804>`_
+
+* Version changes should invalidate pytest runs. (#8790)
+  `PR #8790 <https://github.com/pantsbuild/pants/pull/8790>`_
+
+* Fix pantsbuild wheel publishing. (#8782)
+  `PR #8782 <https://github.com/pantsbuild/pants/pull/8782>`_
+
+* Revert "Upgrade to Pex 2.0.3. (#8704)" (#8787)
+  `PR #8704 <https://github.com/pantsbuild/pants/pull/8704>`_
+  `PR #8787 <https://github.com/pantsbuild/pants/pull/8787>`_
+
+* Add parent-build-id option to know what runs are inner runs and not to run them with pantsd (#8724)
+  `PR #8724 <https://github.com/pantsbuild/pants/pull/8724>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* V2 console UI: Logging improvements (#8806)
+  `PR #8806 <https://github.com/pantsbuild/pants/pull/8806>`_
+
+* Rename V2 goals to use the same naming scheme (#8774)
```

Comment:
This is an API change for anyone consuming V2 rules as a user. 

Why do I recommend moving this to API changes yet recommend moving the new rule into this section? Users of V2 rules vs. authors of rules. I think the former is worth marking API changes, whereas the latter is small enough and experimental enough it can be classified as an internal change.

Reply:
I think both pools are of order of size 1, but I won't fight.